[2024-05-01T01:34:44.820+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2024-05-01T01:02:21.198359+00:00 [queued]>
[2024-05-01T01:34:44.848+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2024-05-01T01:02:21.198359+00:00 [queued]>
[2024-05-01T01:34:44.851+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-01T01:34:44.852+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2024-05-01T01:34:44.853+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-01T01:34:44.894+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-05-01 01:02:21.198359+00:00
[2024-05-01T01:34:44.907+0000] {standard_task_runner.py:55} INFO - Started process 2093 to run task
[2024-05-01T01:34:44.911+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2024-05-01T01:02:21.198359+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpru8k3zc3']
[2024-05-01T01:34:44.913+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask create_table
[2024-05-01T01:34:45.010+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2024-05-01T01:02:21.198359+00:00 [running]> on host fe27111d44a9
[2024-05-01T01:34:45.120+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-05-01T01:02:21.198359+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-01T01:02:21.198359+00:00
[2024-05-01T01:34:45.136+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-05-01T01:34:45.142+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2024-05-01T01:34:45.145+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-05-01T01:34:45.158+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240501T010221, start_date=20240501T013444, end_date=20240501T013445
[2024-05-01T01:34:45.207+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-01T01:34:45.236+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
